cumulusci/core/flows.py

```diff
@@ -117,6 +117,8 @@ def _check_infinite_flows(self, steps, flows=None):
         for step in steps.values():
             if "flow" in step:
                 flow = step["flow"]
+                if flow == "None":
```

Comment:
should probably at the very least check case insensitively, and i'd consider also supporting blank string and yaml !!null.

Reply:
Fair, but I was just aiming to make it compatible with the existing conditional on line 151